packages/components/src/components/Header/Header.js

```diff
@@ -28,14 +27,20 @@ function Header({
   logoutButton,
   onHeaderMenuButtonClick
 }) {
+  function skipToContentClick(event) {
```

Comment:
This can be moved above, outside the component function since it doesn't rely on any state or props.